src/buildercore/core.py

```diff
@@ -678,6 +678,7 @@ def hostname_struct(stackname):
     if domain:
         updates['project_hostname'] = subdomain + "." + domain
         updates['full_hostname'] = hostname + "." + domain
+        updates['ext_node_hostname'] = hostname + "--%s." + domain
```

Comment:
we should have an example I think: `... # "prod--journal--1.elifesciences.org"`

Reply:
Have documented in `docs/dns.md`, I'd leave this as internal unless we document the `context` format